picard/oauth.py

```diff
@@ -88,27 +146,26 @@ def get_authorization_url(self, scopes):
 
     def set_refresh_token(self, refresh_token, scopes):
         log.debug("OAuth: got refresh_token %s with scopes %s", refresh_token, scopes)
-        config.persist["oauth_refresh_token"] = refresh_token
-        config.persist["oauth_refresh_token_scopes"] = scopes
+        self.refresh_token = refresh_token
+        self.refresh_token_scopes = scopes
 
     def set_access_token(self, access_token, expires_in):
         log.debug("OAuth: got access_token %s that expires in %s seconds", access_token, expires_in)
-        config.persist["oauth_access_token"] = access_token
-        config.persist["oauth_access_token_expires"] = int(time.time() + expires_in - 60)
+        self.access_token = access_token
+        self.access_token_expires = int(time.time() + expires_in - 60)
 
     def set_username(self, username):
         log.debug("OAuth: got username %s", username)
-        config.persist["oauth_username"] = username
+        self.username = username
```

Comment:
We don't really need `set_username` if it is only a wrapper around the property.

Reply:
I kept it for now, because it is used for logging.

About logging, i think it should be disabled by default for OAuth, i'm not confortable with OAuth-related stuff being published along debug log. What do you think ?

Reply:
Agreed. We should probably not entirely disable logging, but stop logging details like tokens and probably even username.